akka-docs/src/test/java/jdocs/stream/operators/SourceOrFlow.java

```diff
@@ -355,7 +355,7 @@ void filterExample() {
         Source.from(
             Arrays.asList(
                 ("Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt "
-                        + "ut labore et dolore magna aliqua.")
+                        + "ut labore et dolore magna aliqua")
```

Comment:
I remove this trailing dot because it makes `aliqua` a 7 letter word and it's confusing to see it in the output.